pythonforandroid/toolchain.py

```diff
@@ -58,8 +59,7 @@ def check_python_dependencies():
                 ok = False
 
     if not ok:
-        print('python-for-android is exiting due to the errors above.')
-        exit(1)
+        print('python-for-android is exiting due to the errors logged above')
```

Comment:
Should there be an error raised here? Or is this change intended?

Reply:
I originally did, but actually this code is run before most of the build process (because it has to run before the imports) so I think it needs a different treatment.

Reply:
well but currently, this seems to be a big change in behavior since the code flow just seems to resume. Is that intended? Maybe the `exit` should be left in if a runtime error doesn't make sense here? Or will this problem of `not ok` be evaluated again & properly dealt with later?

Reply:
I also added the exit(1) back in with the previous comment, I missed that I'd accidentally removed it before so thanks for pointing it out!

Reply:
Ah right, the removing line is gone now. Didn't see that :laughing: